ODLA/platforms/odla_popart/odla_compute.cc

```diff
@@ -157,11 +157,11 @@ odla_status odla_ExecuteComputation(odla_computation comp, odla_context context,
   // Config StepIO
   std::map<popart::TensorId, popart::IArray&> inputs;
   for (auto& input : comp->inputs) {
-    inputs.emplace(input.first, *input.second);
+    inputs.at(input.first) = std::move(*input.second);
```

Comment:
"at" returns a reference to the mapped value of the element with key equivalent to key. If no such element exists, an exception of type std::out_of_range is thrown.

Reply:
yes, use at operator will cause exception when key not found, after thinking, there is no case that exist the same tensor_id in odla_ExecuteComputation(), so remove those change